runtime-testsuite/test/org/antlr/v4/test/runtime/descriptors/ParserErrorsDescriptors.java

```diff
@@ -414,7 +414,7 @@
 
 		/**
 		 grammar T;
-		 a : 'a' 'b'* ;
+		 a : 'a' 'b'* EOF ;
```

Comment:
:memo: The original behavior of the test was retained by ensuring the `'b'*` loop didn't look all the way to the end of `a`.